src/vector/index.html

```diff
@@ -49,7 +49,7 @@
         <% }
     } %>
   </head>
-  <body style="height: 100%;" data-vector-indexeddb-worker-script="<%= htmlWebpackPlugin.files.chunks['indexeddb-worker'].entry %>">
+  <body style="height: 100%; margin: 0;" data-vector-indexeddb-worker-script="<%= htmlWebpackPlugin.files.chunks['indexeddb-worker'].entry %>">
```

Comment:
a bit unclear what this change is doing, but probably fine

Reply:
its fixing the margin around the ErrorView and CompatibilityView in Chrome which has a default margin for when our react-sdk css doesn't get loaded yet.